cms/templates/js/settings-tab-section.underscore

```diff
@@ -0,0 +1,10 @@
+<ul class="settings-tab">
+    <li class="settings-section">
+        <a id="general_settings" class="settings-tab-link" href="#"><%- gettext('General Settings') %></a>
```

Comment:
Please use hyphens in class and id names, so "general-settings" not "general_settings".